app/src/main/res/layout/custom_home_carousel_item.xml

```diff
@@ -32,7 +32,6 @@
             android:scaleType="centerCrop" />
 
         <android.support.constraint.ConstraintLayout
-            style="@style/ItemDetailInfo"
```

Comment:
This shouldn't have been removed

Reply:
I saw that, and thought it was moved to the new layout?

Reply:
[here](https://github.com/azavea/cac-tripplanner-android/pull/150/files#diff-1c7694c2931e70d94de62af4124504f2R30)

Reply:
That's for the detail activities - this layout is for the home activity, and shouldn't have been changed